ztpserver/controller.py

```diff
@@ -218,7 +218,7 @@ def put_config(self, request, **kwargs):
 
         fobj = None
         filename = self.expand(node_id, STARTUP_CONFIG_FN)
-        body = str(request.body)
+        body = request.body.decode("utf-8")
```

Comment:
Is this change required?

Reply:
Hi,
It's not related to the model filter, but it's a bugfix to the backup of the switches configurations. This bug was probably introduced with the python3 migration due to the default usage of unicode.
Without this fix, the startup-config is saved as a one-line file when you post the startup-config from the switch to the ztpserver. The saved file wasn't re-usable to restore the swich configuration.

Reply:
I've created a separate PR #407 to fix this